lib/Customer.php

```diff
@@ -78,192 +74,199 @@ public function deleteDiscount($params = null, $opts = null)
         $this->refreshFrom(['discount' => null], $opts, true);
     }
 
+
+    const PATH_BALANCE_TRANSACTIONS = '/balance_transactions';
+
     /**
-     * @param string $id The ID of the customer on which to create the source.
+     * @param string $id The ID of the customer on which to create the customer balance transaction.
      * @param array|null $params
      * @param array|string|null $opts
      *
      * @throws \Stripe\Exception\ApiErrorException if the request fails
      *
-     * @return ApiResource
+     * @return BalanceTransaction
```

Comment:
[blocker] this should be CustomerBalanceTransaction right?
Also this makes me realize that `TaxId` the resource should really be `CustomerTaxId` instead (todo for later)